sdk/src/settings.rs

```diff
@@ -130,6 +141,7 @@ impl Default for Trust {
                 trust_anchors: None,
                 trust_config: None,
                 allowed_list: None,
+                signer_info: None,
```

Comment:
This is not part of trust.  Is part of manifest generation so I would think Builder is the place for this.